### PR TITLE
4.05 changes -- compatibility remarks

### PR DESCRIPTION
**Pull request by gasche**
Followup of #1239. Sorry for the PR noise.

### CODE REVIEW

File: Changes [hunk 1]
```diff
@@ -171,9 +175,11 @@ OCaml 4.05.0 (TBD):
   ListLabels, MoreLabels, StringLabels so they are compatible with
   non-labeled counterparts. Also add missing @@ocaml.deprecated attributes
   in StringLabels and BytesLabels.
-  (Roma Sokolov)
+  (Roma Sokolov, review by Gabriel Scherer, Jacques Garrigue,
+   Gabriel Radanne, Alain Frisch)
```
Comment (jberdine) [inline Changes]: I'm not sure if there is a policy, but note that some multi-line credits entries align under the `(` instead of after the `(` as here.

Comment (gasche) [reply Changes]: Indeed you are a pretty-printing expert :-)

Comment (gasche) [reply Changes]: It looks like there is no rule so far (I started doing the change you suggest, but I found other site that 1-indent, possibly of my past own making), so I'll just leave as it is for now. Thanks for the attention anyway!

Comment (damiendoligez) [reply Changes]: I don't think we should waste too much time on this point.

**Review by damiendoligez — APPROVED** [review]
Looks good. You'll need to rebase before you merge.

**Pull request merged**